sensor_tag_read_block.py

```diff
@@ -67,15 +63,13 @@ class SensorTagRead(Block):
     device_info = ListProperty(SensorTagInfo, title="Sensor Tag Config")
     default_metadata = ObjectProperty(SensorTagMeta)
     scan_length = IntProperty(title="BLE Scan Length", default=5)
-    
+
     def __init__(self):
         super().__init__()
         self._scanner = LEScanner('SensorTag', 5)
         self._configs = {}
         self._tags = {}
-        self._sensors = {}
```

Comment:
did we have support for multiple sensors and we no longer do? What did this do before?